### PR TITLE
third task be

### PR DESCRIPTION
**Pull request by olenastarchak**
Main scope is done:

function getProductsList - https://wdzdjsytsh.execute-api.eu-west-1.amazonaws.com/dev/products

function getProductsById - https://wdzdjsytsh.execute-api.eu-west-1.amazonaws.com/dev/products/909af13e-cfa5-4025-9daf-402697dd7bb9

FE cloudfront - https://d2x4npcbrxn1bb.cloudfront.net/

FE MR - https://github.com/olenastarchak/nodejs-aws-fe/pull/2

Additional scope is done:

Async/await is used in lambda functions
ES6 modules are used for product-service implementation
Webpack is configured for product-service
SWAGGER documentation is created for product-service - https://app.swaggerhub.com/apis-docs/olenastarchak/product-service/0.1
Lambda handlers are covered by basic UNIT tests
Lambda handlers code is written not in 1 single module (file) and separated in codebase
Main error scenarious are handled by API

### CODE REVIEW

File: product-service/handlers/getProductById.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    const product = productList.find(product => product.id === productId);
+
+    return {
+      headers: {
+        'Access-Control-Allow-Origin': '*'
+      },
+      statusCode: product ? 200 : 404,
+      body: product ? JSON.stringify(product) : 'Product not found',
+    };
+  } catch (error) {
```
Comment (Paul044) [inline product-service/handlers/getProductById.js:16]: What would be if axios response fails? Probably, you should return 500 error in catch

File: product-service/handlers/getProducts.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+    return {
+      headers: {
+        'Access-Control-Allow-Origin': '*'
+      },
+      statusCode: 200,
+      body: JSON.stringify(productList)
+    }
+  } catch (error) {
+    console.log(error);
```
Comment (Paul044) [inline product-service/handlers/getProducts.js]: same

File: product-service/tests/getProductById.test.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+/**
+ * @jest-environment node
+ */
+
+import getProductById from '../handlers/getProductById';
+
+describe('getProductsById', () => {
```
Comment (Paul044) [inline product-service/tests/getProductById.test.js:16]: Your test will make actual http requests as you haven't mocked axios

File: product-service/tests/getProductById.test.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  it('should return object of product', async () => {
+    const response = await getProductById({
+      pathParameters: { productId: '909af13e-cfa5-4025-9daf-402697dd7bb9' }
+    })
+    const product = JSON.parse(response.body)
+    expect(product).toBeDefined()
```
Comment (Paul044) [inline product-service/tests/getProductById.test.js]: strange check

File: product-service/tests/getProducts.test.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+/**
+ * @jest-environment node
+ */
+
+import getProducts from '../handlers/getProducts.js';
+
+describe('getProducts', () => {
+  it('should return an array of products', async () => {
+    const response = await getProducts()
```
Comment (Paul044) [inline product-service/tests/getProducts.test.js]: same

File: product-service/handlers/getProductById.js (hunk 1 above)
Comment (olenastarchak) [reply product-service/handlers/getProductById.js:16]: hi! thanks for the comments, I`ll fix asap

**Pull request merged**